conf/modules/stabilization_adaptive_fw.xml

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="stabilization_adaptive" dir="stabilization">
+  <doc>
+    <description>
+      Adaptive attitude and lateral (heading) control for fixedwing aircraft.
+      Mostly based on PID and optional reference generators and feedforward gains.
+      Adaptive parameters should not be used for normal operation at the moment.
```

Comment:
There is a kind of low pass filter to adapt the feedforward gains in flight. That's why it was called 'adaptive' in the first place. But it wasn't really working when I did this years ago and never touched it again...
I can remove the line if you think it is too much ambiguous.